source/Grid/defaultCellRangeRenderer.js

```diff
@@ -40,7 +97,8 @@ export default function defaultCellRangeRenderer ({
         isScrolling,
         key,
         rowIndex,
-        style
+        style,
+        visibility: findOverlap(gridDimensions, style)
```

Comment:
> 1. What does visible mean?
Does it mean the item is fully displayed? Partly displayed? In the use case where a dev may want to render just the "cheaper" parts of the cell, they need to know how much of the cell will be visible, as they will render things differently if 0.03 is visible vs if 0.98 is visible.

I disagree that that information is useful. Let's say you have a cell with an image in the top/left corner and text that wraps around it. If the top/left is 0.05 visible (using the code in this PR) you would need to load the image. If the bottom/left, top/right, or bottom/right corners were 0.05 visible you would not need to load it. In other words, the information is more specific than is useful.

I feel pretty strongly that- if we are to add this feature- it should be a simple boolean. `isVisible` would be `true` if any part of the cell shows at all. If it's an over-scanned row then it is, by definition/previous-implementation, not visible. This would make the calculation very performant.

> Thinking about this more, we could even deprecated (but not remove, for backwards compatibility) isScrolling in favor of visibility and set the state to scrolling during scrolling, so like

Perhaps for a version 9, I could get behind a `visibility` enum [visible, overscan, scrolling] but v9 is (hopefully) a ways off. 😉